src/Parser/Priest/Holy/Modules/Spells/Sanctify.js

```diff
@@ -43,13 +47,22 @@ class Sanctify extends Analyzer {
 
   statistic() {
     const report = this.report;
+    const noHits = (this.casts * MAX_HITS_PER_CAST)- this.totalHits;
+    const hitsPerCast = this.totalHits / this.casts;
 
     return (
       <StatisticBox
         icon={<SpellIcon id={SPELLS.HOLY_WORD_SANCTIFY.id} />}
         value={`${report.sancAvgHits.toFixed(2)}`}
-        label="Average hits"
-        tooltip={`A measure of how many targets were effectively healed by your Holy Word: Sanctify. Over 80% overhealing on a hit is considered a "miss". You missed ${report.sancMissedHits} of ${this.casts * 6} potential hits.`}
+        label="Avg effective hits per cast"
+        tooltip={`A measure of how many targets were effectively healed by your Holy Word: Sanctify. </br>
+                  Over 80% overhealing on a hit is considered a "miss". </br>
+                  <b>Total players hit:</b> ${this.totalHits} </br>
+                  <b>Total casts:</b> ${this.casts} </br>
+                  <b>Hits per cast :</b> ${hitsPerCast.toFixed(2)} </br>
+                  <b>Players overhealed for 80%+ hit:</b> ${this.overhealHit}</br>
+                  <b>Total of no hits:</b> ${noHits}`}
```

Comment:
The wording of this is awkward, but i'm not really sure what to call it. However, it should be at least clarified what a "no hit" is (number of potential hits that didn't have targets and thus didn't hit?) up where you define what a "miss" is.